content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HPE GreenLake Console's API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Secured Yet Agile
+
+One major principle of Application Programming Interface (API) for the HPE Data Services Cloud Console (DSCC) is secured. However, the API must also be flexible for the consumption of applications or tools that use this API to extend their features using DSCC. To accommodate the flexibility, DSCC REST API is using the OAuth 2.0 authentication flow based on the client-credential which generates the limited lifetime access token. This access token will then be embedded in the header of each REST API request as the authorization bearer.
```

Comment:
Hi Ron. I know I reviewed a very similar text block for you recently, but I don't remember exactly where it was. I feel as if it was in this blog post, because the title is the same. Secure, Yet Agile (I remember suggesting that comma as well).

But I don't see my comments incorporated here, so I'll try to put them here for you.
Please change the subtitle here to "Secure, Yet Agile"
Change this to read:
A major guiding principle in the creation of the Application Programming Interface (API) for Data Services Cloud Console (DSCC) from Hewlett Packard Enterprise (HPE) is security. However, to be able to be used by applications or tools that rely on the API to extend their features using DSCC, the API must also be flexible. To provide both security and flexibility, the DSCC REST API uses the 0Auth 2.0 authentication flow based on the client credential, which generates a limited lifetime access token. This access token is then embedded in the header of each REST API request asd the authorization bearer.